pyproject.toml

```diff
@@ -1,69 +1,88 @@
-[tool.poetry]
+[project]
+authors = [
+    {name = "Cloud Custodian Project"},
+]
+license = "Apache-2.0"
+requires-python = "<4.0.0,>=3.9.2"
```

Comment:
Were we removing python 3.9?

Reply:
yes, its causes an issue due to the differential dependency set for c7n-left which requires 3.10+  3.9 is end of life in October, so we're just moving forward with it here, also will note thats in the pr description re changes.